tests/dataset_tests/splitters_tests/test_scaffold_splitter.py

```diff
@@ -0,0 +1,106 @@
+import os
+
+import numpy
+import pytest
+
+
+from chainer_chemistry.dataset.parsers.csv_file_parser import CSVFileParser # NOQA
+from chainer_chemistry.dataset.preprocessors import NFPPreprocessor
+from chainer_chemistry.dataset.splitters.scaffold_splitter import ScaffoldSplitter # NOQA
+from chainer_chemistry.datasets.numpy_tuple_dataset import NumpyTupleDataset
+
+
+@pytest.fixture
+def dataset():
+    here = os.path.abspath(os.path.dirname(__file__))
+    file_rel_path = '../../../examples/own_dataset/dataset.csv'
```

Comment:
We should not use file in example, instead we should

1. prepare file only for testing and use it
2. prepare dataset in other way.

In this case, I think we can prepare `dataset` by `SmilesParser` or `DataFrameParser` to avoid using file.

Reply:
(If possible, it is also preferrable to not to depend on other specific modules like `NFPPreprocessor`, instead more general/basic one is preferrable like `AtomicNumberPrerpcessor` )